### PR TITLE
[Docs] README edits and add CONTRIBUTING file

### PR DESCRIPTION
**Pull request by jacknagz**
### Background

Edits and layout revisions to the README, along with adding the `CONTRIBUTING.md` from the `pounce` repo. We will have to update the install instructions again once we publish to SAR, since the deployment can be simplified.

### CODE REVIEW

**Comment by kostaspap** [comment]
> ### Background
> Edits and layout revisions to the README, along with adding the `CONTRIBUTING.md` from the `pounce` repo. We will have to update the install instructions again once we publish to SAR, since the deployment can be simplified.

SAR app is public. 
Screen Shot 2019-08-29 at 1 04 09 PM

**Comment by kostaspap** [comment]
> ### Background
> Edits and layout revisions to the README, along with adding the `CONTRIBUTING.md` from the `pounce` repo. We will have to update the install instructions again once we publish to SAR, since the deployment can be simplified.

I have tested in my account. It is possible to deploy the SAR app following these steps: 
1. Create CF change set
```
aws serverlessrepo create-cloud-formation-change-set --application-id arn:aws:serverlessrepo:us-east-1:349240696275:applications/aws-remediations --stack-name remediations --capabilities CAPABILITY_NAMED_IAM
```
Keep note of the 'StackId' in the output. 
```
{
    "StackId": "arn:aws:cloudformation:us-west-2:415773754570:stack/serverlessrepo-remediations/9a609af0-ca69-11e9-aba1-02ee71065ed8",
    "ApplicationId": "arn:aws:serverlessrepo:us-east-1:349240696275:applications/aws-remediations",
    "ChangeSetId": "arn:aws:cloudformation:us-west-2:415773754570:changeSet/a4e6c75e0-15dc-460c-a61f-c3343b1e1903/9b4d47a5-cf09-47a1-b2d0-352fb465403f",
    "SemanticVersion": "0.1.0"
}
```

Next step is to execute the change set: 
```
aws cloudformation  execute-change-set --change-set-name <change-set-id>
```
where `change-set-id` is the output of the first command.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Contributing
+
+Before contributing to this repository, please first discuss the change you wish to make by creating an [issue](https://github.com/panther-labs/aws-remediations/issues).
+
+Please note we have a [Code of Conduct](https://github.com/panther-labs/aws-remediations/blob/master/CODE_OF_CONDUCT.md), please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a
```
[the remark this answers is not in the document]
Comment (jacknagz) [reply CONTRIBUTING.md]: It was copy pasta from the Github template IIRC

**Comment by jacknagz** [comment]
Hey @kostaspap, just made some more changes, take a look when you have a sec 😄

File: template.yml [hunk 2]
```diff
@@ -42,7 +27,28 @@ Parameters:
...
+    AllowedValues:
+      - 'true'
+      - 'false'
+  MasterAccountId:
+    Type: String
+    Default: ''
+    Description: >
+      The AWS Account Id where the aws-remediation Lambda exists.
+      This property will only be used if 'IsMasterAccount' parameter is set to 'false'
+  RemediationRoleName:
```
Comment (kostaspap) [inline template.yml]: We rely on this to be `AwsRemediationRole` ( https://github.com/panther-labs/aws-remediations/blob/master/src/app/remediation_base.py#L111 )
If you make this configurable, you have to pass it in the Lambda as a parameter. 

We could make this change in a separate PR as well (make configurable + pass as parameter), i don't think it's urgent the change

File: template-sar.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+AWSTemplateFormatVersion: 2010-09-09
+Transform: AWS::Serverless-2016-10-31
+Description: Instantly remediate common security issues in your accounts
+
+Resources:
+  PantherAWSRemediations:
```
Comment (kostaspap) [inline template-sar.yml]: Maybe remove the "Panther" prefix from here, given it's open source project

File: template.yml (hunk 2 above)
Comment (jacknagz) [reply template.yml]: I added it because there's a chance there could be a naming conflict in an account, I experienced this first-hand

**Comment by jacknagz** [comment]
FYI @kostaspap the conditional change is due to:

```
$ cfn-lint template.yml
W1001 Ref to resource "Function" that may not be available when condition "CreateSSM" is True and when condition "IsMasterAccount" is False at Resources/InvokeRemediation/Properties/Content/mainSteps/0/inputs/FunctionName/Ref
template.yml:154:15
```
